WordPress/gutenberg
# Widget Editor: Don't disable the Save button

Similar to #42842

## What?

PR removes the `disabled` attribute from the Widget Editor's save button component.

![widget-editor-save-button](https://github.com/WordPress/gutenberg/assets/54422211/bc08f97a-818f-402c-8a69-d23b6d53c2c7)

## Why?

#42802 about the Site editor explains why having both disabled and aria-disabled is a problem.

## How?

Moved the variable that defines whether the `disabled` attribute is enabled to the `aria-disabled` attribute and removed the `disabled` attribute.

## Testing Instructions

- Activate TT1 theme.
- Access Appearance > Widgets.
- Open the browser's developer tools.
- The save button should not have `disabled` attribute and `aria-disabled` should be `true`.
- Make changes to the content.
- The save button should not have `disabled` attribute and `aria-disabled` should be `false`.
- Click the save button.
- The save button should not have `disabled` attribute and `aria-disabled` should be `true`.


## Screenshots or screencast 

https://github.com/WordPress/gutenberg/assets/54422211/1459788c-2394-4c5a-88a5-1029d4ffe620
